src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/localize/templatestrings.en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "author": "Microsoft",
+  "name": "Razor View",
+  "description": "A Razor view without a page model",
```

Comment:
Let's review these strings with Sayed as well.

Reply:
yup this is more of a POC, won't be actually merging this PR, just a quick sample